src/controllers/site.controller.js

```diff
@@ -2,46 +2,35 @@
 
 const Constants = require('../constants')
 const BaseController = require('./base.controller')
+const SiteValidator = require('../validators/site.validator')
 
 module.exports = class SiteController extends BaseController {
-  static async doGet (request, reply, errors = undefined) {
+  static async doGet (request, reply, errors) {
     try {
-      const context = {
-        pageTitle: 'Waste Permits - Site',
-        message: 'Hello, World!',
-        errors: errors
-      }
+      const pageContext = BaseController.createPageContext('What\'s the site name?', errors, SiteValidator)
```

Comment:
If anything needs escaping in a string, I prefer to use double quotes for the string rather than specifically escaping the character e.g. `"What's the site name?"`

Do you have an opinion @irisfaraway ?

Reply:
Using double quotes goes against StandardJS and triggers a warning. Can we use ` instead?

Reply:
I would have gone with the double quotes too – must just be a Ruby convention. Think I'd prefer ` over the escaping slash.

Reply:
Contrary to my earlier comment, the StandardJS linter only moans about the double quote if the enclosed escape character \ is deemed unnecessary. So we could use double quotes after all, but we have decided to go with ` characters wherever we would otherwise need to escape single quotes or where we are going to carry out complex string manipulations.